### PR TITLE
[JFG-860 Embedded Tomcat for web client]

### PR DESCRIPTION
**Pull request by SokolAndrey**
To start web client with embedded Tomcat:
Build web client project using mvn;
Run java -jar target/standalone.jar [-extractDirectory] [-resetExtract]

### CODE REVIEW

**Comment by dlatnikov** [comment]
please, add start command to the page below:
https://github.com/griddynamics/jagger/blob/master/doc/pages/WebUiInstallation.txt

smth like:
"Starting from commit XXX, to start WebUI please do: ..."

don't delete old instruction

also I would like to know how to change WebUI properties

File: webclient/pom.xml [hunk 1]
```diff
@@ -29,12 +29,6 @@
...
         <dependency>
-            <groupId>com.google.gwt</groupId>
-            <artifactId>gwt-user</artifactId>
-            <version>${gwt.version}</version>
-        </dependency>
-
-        <dependency>
             <groupId>com.sencha.gxt</groupId>
             <artifactId>gxt</artifactId>
             <version>3.0.1</version>
```
Comment (anton-antonenko) [inline webclient/pom.xml]: Please, extract all versions to properties

File: webclient/pom.xml [hunk 2]
```diff
@@ -281,29 +275,28 @@
...
+                    <execution>
+                        <id>tomcat-run</id>
+                        <goals>
+                            <goal>exec-war-only</goal>
+                        </goals>
+                        <phase>package</phase>
+                        <configuration>
+                            <path>/</path>
+                            <enableNaming>false</enableNaming>
+                            <finalName>standalone.jar</finalName>
```
Comment (anton-antonenko) [inline webclient/pom.xml]: Maybe it's better to give more descriptive name to jar? e.g. jagger-web-client.jar

Comment (dlatnikov) [inline webclient/pom.xml]: +1
war had name: webclient-{version}.war

**Comment by dlatnikov** [comment]
is there a chance to set web client properties (DB settings) before or during jar launching. current approach:
- launch war => it is unpacked
- go to unpacked directory and change properties
- relaunch war

it will be great to avoid second relaunching. like:
- set properties
- launch jar

**Comment by a-badaev** [comment]
+1

File: doc/pages/WebUiInstallation.txt [hunk 3]
```diff
@@ -57,6 +57,14 @@
 ///     to write there - redefine path @n
 /// @n
 /// @n
+/// @li @b Note: 26/09/16 Starting from commit 3f1fe46 the Web Client can be run with embedded server. For this
+///        run the file standalone.jar from com/griddynamics/jagger/webclient folder.
+///        java -jar {Route/to/webclient}/target/standalone.jar [-extractDirectory=specify/path] [-resetExtract]
```
Comment (dlatnikov) [inline doc/pages/WebUiInstallation.txt]: doesn't match to <finalName>jagger-${build.finalName}.jar</finalName>

**Comment by dlatnikov** [comment]
👍

**Pull request merged**